app/assets/stylesheets/question_index.scss

```diff
@@ -0,0 +1,25 @@
+.question-body {
+    font-size: larger;
+    font-family: -webkit-body;
+  }
+
+  .button-round {
+    margin-right: 347px;
+    color: black;
+    background-color: aliceblue;
+    padding-left: 30px;
+    padding-right: 30px;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    font-size: large;
+    font-family: serif;
+    border-radius: 10px;
+    border-width: 2px;
+    border-color: blanchedalmond;
```

Comment:
Color `blanchedalmond` should be written in hexadecimal form as `#ffebcd`<br>Color literals like `blanchedalmond` should only be used in variable declarations; they should be referred to via variable everywhere else.